snippets/variant-list-row.liquid

```diff
@@ -0,0 +1,190 @@
+              | escape
+            }}
+          </div>
+        {% endif %}
+      </div>
+    {%- endif -%}
+    <div class="small-hide">
+      <span class="variant-item__name h4 break">{{ item.title | escape }}</span>
+      {%- if section.settings.show_sku -%}
+        <span class="variant-item__sku caption break">{{ sku | escape }}</span>
+      {%- endif -%}
+    </div>
+  </td>
+
+  <td class="variant-item__details medium-hide large-up-hide">
+    <div class="variant-item__info">
+      <span class="variant-item__name h4 break">{{ item.title | escape }}</span>
+      {%- if section.settings.show_sku -%}
+        <span class="variant-item__sku caption break">{{ sku | escape }}</span>
+      {%- endif -%}
+    </div>
+
+    {%- render 'loading-overlay' -%}
+
+    {%- if item.compare_at_price != item.price -%}
+      <dl class="variant-item__discounted-prices">
+        <dt class="visually-hidden">
+          {{ 'products.product.price.regular_price' | t }}
+        </dt>
+        <dd>
+          <s class="variant-item__old-price price price--end">
+            {{ item.compare_at_price | money }}
+          </s>
+        </dd>
+        <dt class="visually-hidden">
+          {{ 'products.product.price.sale_price' | t }}
+        </dt>
+        <dd class="price">
+          {{ item.price | money -}}
+          <span>{{ 'sections.variant_list.each' | t }}</span>
+        </dd>
+      </dl>
+    {%- else -%}
+      <span class="price">
+        {{ item.price | money -}}
+        <span>{{ 'sections.variant_list.each' | t }}</span>
+      </span>
+    {%- endif -%}
+
+    {%- if item.available and item.unit_price_measurement -%}
+      <div class="unit-price caption">
+        <span class="visually-hidden">{{ 'products.product.price.unit_price' | t }}</span>
+        {{ item.unit_price | money }}
+        <span aria-hidden="true">/</span>
+        <span class="visually-hidden">&nbsp;{{ 'accessibility.unit_price_separator' | t }}&nbsp;</span>
+        {%- if item.unit_price_measurement.reference_value != 1 -%}
+          {{- item.unit_price_measurement.reference_value -}}
+        {%- endif -%}
+        {{ item.unit_price_measurement.reference_unit }}
+      </div>
+    {%- endif -%}
+  </td>
+
+  <td class="variant-item__totals right medium-hide large-up-hide">
+    {%- render 'loading-overlay' -%}
+    <span class="price">Placeholder</span>
+  </td>
+  <td class="variant-item__quantity">
+    <div class="variant-item__quantity-wrapper">
+      <label class="visually-hidden" for="Quantity-{{ variant.id }}">
+        {{ 'products.product.quantity.label' | t }}
+      </label>
+      <div class="variant-item__quantity-info">
+        {% render 'icon-info' %}
+      </div>
+      {% render 'quantity-input', variant: variant %}
+      {%- if cart_qty > 0 -%}
+        <variant-list-remove-button
+          id="Remove-{{ variant.id }}"
+          data-index="{{ variant.id }}"
+        >
+          <a
+            href="{{ item.url_to_remove }}"
```

Comment:
I think this url is doing nothing in our case. I see that this was copied from the cart where we iterate through items. 

1. `Items` in the cart is an array of `line_items`.  [source](https://shopify.dev/docs/api/liquid/objects/cart#cart-items)
2. `line_item` has a parameter `url_to_remove`. [source](https://shopify.dev/docs/api/liquid/objects/line_item#line_item-url_to_remove)
3. So, it works perfectly in the cart. 

In our case we iterate through variants and `variant` doesn't have a parameter `url_to_remove`.